tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -1296,3 +1296,20 @@ def test_read_parquet_with_sql_simplifier(session):
         .select((col("num") + 1).as_("num"))
     )
     assert df2.queries["queries"][-1].count("SELECT") == 4
+
+
+def test_filepath_not_exist_or_empty(session, local_testing_mode):
```

Comment:
This test is not enabled for Local Testing (those are marked with `@pytest.mark.localtest`). You shouldn't need `local_testing_mode`.